### ISSUE TITLE
Windows search Stops working even if only Essential Tweaks are applied in windows 10

### ISSUE THREAD

**Issue by Shubham20091999** [ChrisTitusTech/win10script#253]
Windows search stops working even if just Essential tweaks are applied in windows 10.
Previously This was happening only when I try to disable cortana(search)

**Comment by FlexMcMurphy** [comment]
I added it back by right clicking on the Taskbar then Search > Show Search Box.

**Comment by Shubham20091999** [comment]
> I added it back by right clicking on the Taskbar then Search > Show Search Box.

In my case Windows Search just stop detecting programs installed after running essential tweaks. Programs installed before the tweaks is showing up in search but not the ones installed after the tweaks.

**Comment by FlexMcMurphy** [comment]
Yes I have that problem also. I installed Notepad++ after doing the Essential Tweaks but then searching for it in Windows Search did not find it. It is actually installed and I can get it from the start menu.

**Comment by Shubham20091999** [comment]
> Yes I have that problem also. I installed Notepad++ after doing the Essential Tweaks but then searching for it in Windows Search did not find it. It is actually installed and I can get it from the start menu.
Also you can add one random folder in indexing and add all the shortcuts to the program you want in there. Windows search will pick it up. but you might loose some functionality, like in the case of Firefox if this essential tweaks are not applied you can start a private browser directly from start menu but you cant do this if you add the shortcut in indexing. I don't know why this is happening but I just noticed it.

**Comment by kasai2210** [comment]
You need to again run the script and now this time, click on the `Enable Background Apps` button to enable background apps, that might fix your issue.

**Comment by damian101** [comment]
Yes, disabling background apps seems to be the issue, it should not be done by something called "Essential Tweaks" imo.
